fall18/p4/README.md

```diff
@@ -1,3 +1,301 @@
 # Project 4
 
-More details coming soon.
+The project focuses on the IMDB movies dataset that contains a list of movies along with information like cast, director, rating, genre, plot description etc. You can open the IMDB-Movie-Data file in Excel or some other software to get an idea of what the dataset contains.
```

Comment:
This project is for all you cinema enthusiasts out there! This week, we will be looking at data set about  ~1,000 most popular movies on IMDB in the last 10 years. Each movie in the dataset has like cast, director, rating, genre, runtime, budget, etc.